error/src/lib.rs

```diff
@@ -252,6 +252,10 @@ impl CaliptraError {
 
     /// FMC Errors
     pub const FMC_RT_ALIAS_UNIMPLEMENTED: CaliptraError = CaliptraError::new_const(0x000F0001);
```

Comment:
move to the end of the list instead of the beginning so we don't leave a hole once we implement this code and remove this error